pool/src/lib.rs

```diff
@@ -65,6 +80,77 @@ impl LendingPoolTrait for LendingPool {
         Ok(())
     }
 
+    /// Configures the reserve collateralization parameters
+    /// all the values are expressed in percentages with two decimals of precision.
+    ///
+    /// # Arguments
+    ///
+    /// - asset The address of asset that should be set as collateral
+    /// - config Collateral parameters
+    ///
+    /// # Panics
+    ///
+    /// - Panics with `InvalidReserveParams` when wrong collateral params provided.
+    /// - Panics if the caller is not the admin.
+    fn configure_as_collateral(
+        env: Env,
+        asset: Address,
+        params: CollateralParams,
+    ) -> Result<(), Error> {
+        Self::ensure_admin(&env)?;
+
+        //validation of the parameters: the LTV can
+        //only be lower or equal than the liquidation threshold
+        //(otherwise a loan against the asset would cause instantaneous liquidation)
+        assert_with_error!(
+            &env,
+            params.ltv < params.liq_threshold,
```

Comment:
Comment above states that the LTV can only be lower **or equal** to the liquidation threshold, but I see that you validate only the `less than` (<).